lib/scsi_defs.h

```diff
@@ -108,5 +112,18 @@ namespace scsi_defs {
     kCompleted = 0xF
   };
 
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6 Table 58
+  // The fields are arranged according to Big Endian format.
```

Comment:
Since we decided to just have one comment about Big Endian, can we move this comment to the top of all structs or something?

Reply:
Yes, we can put this at the top of the file. Making the change right now.